_posts/en/newsletters/2023-08-09-newsletter.md

```diff
@@ -0,0 +1,333 @@
+  node to abuse the mechanism to collect fees from its counterparties.
+  She described some challenges that would be faced by a mechanism like
+  the one Towns sketched and he [replied][towns dos2] with counterpoints
+  and a summary: "I think monetary-based DoS deterrence is still likely
+  to be a fruitful area for research if people are interested, even if
+  the current implementation work is focused on reputation-based
+  methods."
+
+- **HTLC endorsement testing and data collection:** Carla Kirk-Cohen and
+  Clara Shikhelman [posted][kcs endorsement] to the Lightning-Dev
+  mailing list to announce that developers associated with Eclair, Core
+  Lightning, and LND were implementing parts of the [HTLC
+  endorsement][topic htlc endorsement] protocol in order to begin
+  collecting data related to it.  The help this, they're proposing a set
+  of data that will be useful for test nodes to collect for researchers.
+  Many of the fields are intended to have their data randomized to
+  prevent leaking information that could reduce the privacy of spenders
+  and receivers.  They intended for there to be multiple phases of
+  testing and outline how participating nodes will act during the
+  different phases.
+
+- **Proposed changes to Bitcoin Core default relay policy:** Peter Todd
+  started two threads on the Bitcoin-Dev mailing list related to pull
+  requests he's opened to change Bitcoin Core's default relay policy.
+
+  - *Full RBF by default:* the [first thread][todd rbf] and [pull
+    request][bitcoin core #28132] proposes making [full RBF][topic rbf]
+    the default setting in a future version of Bitcoin Core.  Currently,
+    Bitcoin Core by default will only relay and accept into its mempool
+    replacements of unconfirmed transactions if the transaction being
+    replaced contains the [BIP125][] signal indicating opt-in
+    replacability (and if both the original transaction and the
+    replacement transaction follow some other rules), called opt-in RBF.
+    A configuration option, `-mempoolfullrbf`, allows node operators to
+    instead choose to accept replacements of any unconfirmed transaction
+    even if it didn't contain the BIP125 signal, called _full RBF_.
+    Peter Todd's proposal would make full RBF the default but allow node
+    operators to change their settings to choose opt-in RBF instead.
+
+      Peter Todd argues that the change is warranted because (according
+      to his measurements, which have been [called into doubt][towns
+      rbf]) a significant percentage of mining hash rate is apparently
+      following full RBF rules and there are enough relay nodes that
+      have enabled full RBF to allow non-signaled replacements to make
+      it to those miners.  He also says he is unaware of any active
+      business that currently accepts unconfirmed onchain transactions
+      as final payment.
+
+  - *Removing specific limits on `OP_RETURN` outputs:* the [second
```

Comment:
Possibly link to https://developer.bitcoin.org/devguide/transactions.html#null-data for background?

Reply:
That section has an error I'm aware of (and haven't gotten around to fixing), so gonna skip that for now.